### ISSUE TITLE
All the devs working to improve twitter for free:

### ISSUE THREAD

**Issue by swingcake** [twitter/the-algorithm#1459]
![20230402_225301.jpg](https://user-images.githubusercontent.com/32037820/229368937-6155075c-8a38-44a1-af85-265ca5936d3b.jpg)

**Comment by gonzoknows** [comment]
They could be improving mastodon which is fully open source 
https://github.com/mastodon/mastodon

**Comment by swingcake** [comment]
> They could be improving mastodon which is fully open source 
> https://github.com/mastodon/mastodon

..and not owned by a multi-billionaire

**Comment by gonzoknows** [comment]
> > They could be improving mastodon which is fully open source
> > https://github.com/mastodon/mastodon
> 
> ..and not owned by a multi-billionaire

Exactly, they will actually get recognition, unlike twitter where they will just sweep them under the rug

**Comment by MurphyAdam** [comment]
They could literally be lifting and doing tax evasion, but instead they choose this. The west has fallen.
